leaf-ui/js/loadSaveFunctions.js

```diff
@@ -198,21 +181,50 @@ function getFuncSegList(arr) {
 	return res;
 }
 
+*/
```

Comment:
Why are they commented out and not deleted?

Reply:
Later on we might want to take into account of the graphs made before the backbone-refactor, so I thought it was better to keep it.Could definitely move it though

Reply:
Maybe add a TODO so other people know why you kept them.

Reply:
I added the comment.